### PR TITLE
[ID-1330] Migrating chart to Repo.

### PR DESCRIPTION
**Pull request by DandyDeveloper**

### CODE REVIEW

File: chart/gorush/Chart.yaml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+apiVersion: v1
+appVersion: "1.0"
```
Comment (pwhiteside) [inline chart/gorush/Chart.yaml]: should be `latest`, right?

Comment (DandyDeveloper) [reply chart/gorush/Chart.yaml]: Nope, it gets overwritten anyway.

Comment (pwhiteside) [reply chart/gorush/Chart.yaml]: Not for local development

File: chart/gorush/templates/deployment.yaml [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+      labels:
+        app: {{ template "gorush.name" . }}
+        release: {{ .Release.Name }}
+    spec:
+      containers:
+      - env:
+        - name: EEN_DHASH_ADDRESS
+          value: {{ .Values.eenDhashAddress }}
+        - name: EEN_IOS_PROD_CERT
+          value: {{ .Values.secret.iodProdCert }}
```
Comment (EE-GSlomin) [inline chart/gorush/templates/deployment.yaml]: iosProdCert

**Comment by EE-GSlomin** [comment]
Besides the typo 👍

**Pull request merged**